### DISCUSSION TITLE
crates.io is using moka

### DISCUSSION THREAD

**Discussion by Turbo87** [moka-rs/moka#51] [Show and tell]
With https://github.com/rust-lang/crates.io/pull/3999 we've started using `moka` for crates.io and it appears to be working very well. We're seeing cache hit rates of ~85% for the high-traffic download endpoint now.

Thank you for working on this! :)

**Comment by tatsuya6502** [comment]
@Turbo87 — That is a great news for me! Thank you very much for letting me know.
